app/pages/users/groups/group_list.php

```diff
@@ -0,0 +1,21 @@
+<?php
+restrict_access(Access::$EDIT_USERS);
+$groups = em()->getRepository(UserGroup::class)->findAll();
```

Comment:
Prefer DQL with projection here for performance. Not a merge blocker though

Reply:
Done